.gitignore

```diff
@@ -1,5 +1,7 @@
 **/*.svg
 **/*.png
+!**/testdata/*_golden.png
```

Comment:
`!` at the beginning of the line tells git to not ignore golden files. Before this change all *.png files were ignored.